pkg/authn/k8schain/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/google/go-containerregistry/pkg/authn/k8schain
+
+go 1.15
```

Comment:
the parent is 1.14, these should be consistent.

Reply:
I think it would be fine for them to diverge in the future, but I've dropped this to 1.15.